tools/clang/test/DXC/PipelineStage.hlsl

```diff
@@ -0,0 +1,31 @@
+// RUN: %dxc -T ps_6_5 -P -Fi %t.ps_65.hlsl.pp %s
+// RUN: FileCheck --input-file=%t.ps_65.hlsl.pp %s --check-prefix=PS_65
+// PS_65:ps 6 5
+
+// RUN: %dxc -T ps_6_6 -P -Fi %t.ps_66.hlsl.pp %s
+// RUN: FileCheck --input-file=%t.ps_66.hlsl.pp %s --check-prefix=PS_66
+// PS_66:ps 6 6
+
+// VS PS GS HS DS CS LIB MS AS
```

Comment:
With all the stages listed below, was this test intended to test each one, rather than just PS?

Reply:
Nice catch.
Will fix it.